php/class-post-type.php

```diff
@@ -312,4 +374,21 @@ public function add_editor_color_palette() {
 			]
 		);
 	}
+
+	/**
+	 * Admin page post meta query.
+	 *
+	 * @param array  $args    Filter args.
+	 * @param object $request Filter request.
+	 *
+	 * @return array
+	 */
+	public function admin_page_sub_menu_query( array $args, $request ): array {
```

Comment:
Do we still need this?

Reply:
Good spot! Did some testing and definitely don't need it. Everything is working fine without it. Thanks for picking that up.